tests/unit-tests/test_gnn.py

```diff
@@ -74,7 +74,8 @@ def create_rgcn_node_model(g):
                                        num_bases=2,
                                        num_hidden_layers=1,
                                        dropout=0,
-                                       use_self_loop=True)
+                                       use_self_loop=True,
+                                       norm='batch')
```

Comment:
You can check all three cases: norm=None, norm='batch' and norm='layer'

Reply:
Please follow https://github.com/awslabs/graphstorm/blob/main/tests/unit-tests/test_dataloading.py#L294C1-L295 to generate different options for testing.